src/utils/axiosInterceptors.ts

```diff
@@ -22,11 +22,14 @@ declare module 'axios' {
   }
 }
 
-export const registerAxiosCacheRetryInterceptors = (): any => {
+export const registerInitialAxiosInterceptors = (): any => {
   findAndDeleteExpiredCachedItems();
+  // note that the interceptors are called in reverse order in which they are registered - last
+  // defined interceptor is called first:
   axios.interceptors.request.use(logCurl, error => Promise.reject(error));
   axios.interceptors.request.use(fetchCachedResponse, error => Promise.reject(error));
   axios.interceptors.response.use(saveCacheResponse, error => retryRequests(error));
+  // additionally, `registerHostnameReplacing()` registers another request interceptor (if/when called)
```

Comment:
Is this code here a leftover or is some code missing here?

If it's meant to be here, I think it would need to be changed because now it suggests there should be some code after it.

Reply:
It was meant to be here, but apparently it wasn't as clear as I hoped it would be. :)

I have now changed the comments, is this better?

Reply:
Thanks, it is better now :) :+1: 

Resolving